### ISSUE TITLE
Memory leak when using preview feature of VapourSynth Editor

### ISSUE THREAD

**Issue by Kagami** [vapoursynth/vapoursynth#243]
I already reported this issue to @mystery-keeper and he said that you're aware of it. Filing it here for easier tracking.

How to reproduce:

* Type in vsedit some simple script (better to use Full HD source and heavy filters that consume memory a lot for the most dramatic effect). E.g.:

  ```python
import vapoursynth as vs
import havsfunc as haf
core = vs.get_core()
c = core.ffms2.Source('1.ts')
c = haf.QTGMC(c, TFF=True)
c = haf.Deblock_QED(c, quant1=60, quant2=60)
c = core.dfttest.DFTTest(c, sigma=200)
c.set_output()
```

* Measure memory consumption of vsedit (`ps -C vsedit -o rss=` in Linux) in various moments of time.

  ```
Right after opening: 47104
Open preview window first time: 916740
Press preview one more time: 1364520
Press preview 5 more times: 6244060
Close preview window: 6068476
Open preview again: 7176268
```

In other words, vsedit quickly consumes all available memory on PC if you use preview a lot.

Software/filter versions: all latest from git.
Platform: Gentoo Linux x86-64.

**Comment by myrsloik** [comment]
Yes, the curse of the versions with big changes and me being lazy about testing. I'll see if I have time to debug it properly by next week (and then spend lots of time testing)

**Comment by myrsloik** [comment]
Fixed now

**Comment by Kagami** [comment]
I rebuilt latest master, but the problem still remains unfortunately. Memory consumption increases every time I press F5.

**Comment by Kagami** [comment]
And I get warning messages `Core freed but 173 filter instances still exist` in vsedit's log window (which you've just added in 2c50e24ca9a227448ba478655bf52be507762e44).

**Comment by myrsloik** [comment]
This is actually caused by a leak in dctfilter which @mystery-keeper has been informed about

**Comment by Kagami** [comment]
I can confirm, with `c = haf.Deblock_QED(c, quant1=60, quant2=60)` line commented, memory consumption doesn't exceed 1.5 gigabytes no matter how many times I pressed F5.
Thanks for the quick fix!
